### ISSUE TITLE
AttributeError: module 'oyaml' has no attribute 'FullLoader'

### ISSUE THREAD

**Issue by forkbabu** [wimglenn/oyaml#23]
```
data = yaml.load(opt.config_file, Loader=yaml.FullLoader)
AttributeError: module 'oyaml' has no attribute 'FullLoader'
```
yaml.__version__  is 3.13
oyaml is 1.0 latest pypi build.

**Comment by forkbabu** [comment]
!pip install -U PyYAML helped. Bumped package to 5.4.1 
I was doing this in a colab notebook.

**Comment by wimglenn** [comment]
The `FullLoader` class is only available in PyYAML 5.1 and later.
